src/ui/widgets/EmbeddedDisplay/opiParser.ts

```diff
@@ -636,6 +637,7 @@ export const OPI_SIMPLE_PARSERS: ParserDict = {
   textAlignV: ["vertical_alignment", opiParseVerticalAlignment],
   backgroundColor: ["background_color", opiParseColor],
   foregroundColor: ["foreground_color", opiParseColor],
+  emptyColor: ["empty_color", opiParseColor],
```

Comment:
This property is called `empty_color` in bob files, but `color_fillbackground` in opi files. I'd suggest maybe moving this line into the `BOB_SIMPLE_PARSERS` and assigning `color_fillbackground` to `emptyColor` here. This way we can set the empty colour for both bob and opi (at the moment we can only set it for bob files this way).